frontend/src/i18n/locales/types.ts

```diff
@@ -1,7 +1,21 @@
 interface Translation {
   translation: {
-    general: {
-      slogan: string
+    features: {
```

Comment:
wonder if there is a way to automate this... seems like it could be a PITA to update